nonprofit-board-management.php

```diff
@@ -94,6 +94,32 @@ public function load_plugin_textdomain() {
       load_plugin_textdomain( 'nonprofit-board-management', false, dirname( plugin_basename( __FILE__ ) ) . '/languages/' );
     }
 
+	/**
+	 * Add the Board Member user role on plugin activation.
+	 *
+	 * @param boolean $network_wide If the plugin is network activated.
+	 */
+	public function activate_plugin( $network_wide ) {
+
+		/**
+		 * If the plugin is network activated we need to add the
+		 * Board Member role for each subsite.
+		 */
+		if ( is_multisite() && $network_wide ) {
+
+			global $wpdb;
+			$blog_ids = $wpdb->get_col( "SELECT blog_id FROM $wpdb->blogs" );
+			foreach ( $blog_ids as $blog_id ) {
+				switch_to_blog( $blog_id );
+				$this->add_board_roles();
+				restore_current_blog();
+			}
+
+		} else {
+
+			$this->add_board_roles();
+		}
+	}
 
     /*
      * Add the board roles when the plugin is first activated.
```

Comment:
Do you want to update this comment since it still references activation? This function has been separated out from activation so I figure it should be modified.

Reply:
Yeah I can do that. This function is only called on plugin activation but I'll try to clarify. I guess the function name is likely clarification enough.